x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/saml/SamlAuthenticatorTests.java

```diff
@@ -1417,9 +1417,15 @@ private Encrypter getEncrypter(Tuple<X509Certificate, PrivateKey> keyPair) throw
         final Credential keyEncryptionCredential = new BasicCredential(keyPair.v1().getPublicKey(), keyPair.v2());
         KeyEncryptionParameters keyEncryptionParameters = new KeyEncryptionParameters();
         keyEncryptionParameters.setEncryptionCredential(keyEncryptionCredential);
-        keyEncryptionParameters.setAlgorithm(
-            randomFrom(EncryptionConstants.ALGO_ID_KEYTRANSPORT_RSAOAEP, EncryptionConstants.ALGO_ID_KEYTRANSPORT_RSA15)
-        );
+        if(inFipsJvm()){
```

Comment:
this change is probably worth calling out in the public documentation.  Triple DES and RSA 1.5 are notable encryption standards no longer allowed with FIPS.